### DISCUSSION TITLE
Add RaspiAudio devices as player providor

### DISCUSSION THREAD

**Discussion by mchipser** [music-assistant/support#1647] [Feature requests and ideas]
The built in media player in Home Assistant allows the use of RaspiAudio devices as media playback. When using Media Assistant the user is unable to pick these devices as playback devices. 
https://raspiaudio.com/

**Comment by mchipser** [comment]
![image](https://github.com/music-assistant/hass-music-assistant/assets/40067867/cfbc37f2-0d9f-4405-a1be-2ec466b40d7f)

**Comment by mchipser** [comment]
It looks like this is already being worked, as an alternate player provider.. 

From Discord: marcelveldt — 11/20/2023 10:26 AM
Correct, but it will come soon as we're working on support to use any HA media_player as input source for MA

**Comment by UsbBitMinter** [comment]
Any updates on this? this is the only item thats preventing my use of MA :(

**Reply by OzGav** [reply]
Today’s beta93 added support for HA media player entities.
